src/sidebar/services/api.js

```diff
@@ -182,8 +173,8 @@ function createAPICall(
  * API client for the Hypothesis REST API.
  *
  * Returns an object that with keys that match the routes in
- * the Hypothesis API (see http://h.readthedocs.io/en/latest/api/). See
- * `APICallFunction` for the syntax of API calls. For example:
+ * the Hypothesis API (see http://h.readthedocs.io/en/latest/api/).
+ * @see APICall for the syntax of API calls. For example:
```

Comment:
The benefit of `@see` is that Visual Studio Code (and other IDEs) will understand that it is a reference to a symbol and will offer go-to-definition etc. for it.